src/types/gravity/artworkResponse.d.ts

```diff
@@ -0,0 +1,170 @@
+// This is generated from taking a few Artwork responses from
+// Gravity and throwing them into QuickType. You should consider it
+// 90% accurate, but not perfect.
```

Comment:
Neat use of QuickType. Curious how many artworks you fed it before you felt confident in the results?

Reply:
I put in 10 artworks, because I had them in the debugger - just ran `JSON.stringify(work)` and passed it into the website